scripts/generate/templates/Checkpoints.js

```diff
@@ -133,13 +133,6 @@ function getAtProbablyRecentBlock(${opts.historyTypeName} storage self, uint256
     return pos == 0 ? 0 : _unsafeAccess(self.${opts.checkpointFieldName}, pos - 1).${opts.valueFieldName};
 }
 
-/**
- * @dev Returns checkpoint at given position.
- */
-function getAtPosition(History storage self, uint32 pos) internal view returns (Checkpoint memory) {
-    return self._checkpoints[pos];
-}
-
```

Comment:
Why is this removed?

Reply:
My understanding is it was added, not used, and then removed. If you look at the overall PR diff, this file (and the resulting solidity) are not touched

Reply:
re-added